src/components/util/ColorSwatch.jsx

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { readableColor } from 'polished';
```

Comment:
we already use `tinycolor`  in color utilities for dictionary - we could use the [`mostReadable` function](http://bgrins.github.io/TinyColor/docs/tinycolor.html#section-31) instead of adding the `polished` package

Reply:
fun fact, the underlying math for checking color brightness is this:
`($R * 299 + $G * 587 + $B * 114 ) / 1000`

It's a weighted sum that returns 0-255 where 255 is the brightest. This is based on a color space that compensates for how we see green as brighter than other colors on screens.

Reply:
tinycolor does this math slightly differently: `return (0.2126 * R) + (0.7152 * G) + (0.0722 * B);`

it's still roughly a 30/60/10 weight of R/G/B